chapters/equations.tex

```diff
@@ -753,11 +753,17 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value of the \lstinline!start!-attribute as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
 \end{nonnormative}
 
-If a parameter has a modifier for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the modifier for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
-In this case a diagnostic message is recommended in a simulation model.
+A parameter not having \lstinline!fixed = false! and neither having a binding equation nor being part of a record having a binding equation, is handled in different ways depending on the presence of a modifier for the \lstinline!start!-attribute.
+If there is no modifier for the \lstinline!start!-attribute, it is an error unless a parameter value is explicitly provided before solving the initialization problem.
+In this case, a tool may give a diagnostic message when translating a simulation model to inform the user that additional parameter values must be provided later.
+If there is a modifier for the \lstinline!start!-attribute, the modifier can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
```

Comment:
```suggestion
A parameter not having \lstinline!fixed = false! and neither having a binding equation nor being part of a record having a binding equation, is handled in different ways depending on whether its \lstinline!start!-attribute has a value or not.
If there is no value for the \lstinline!start!-attribute, it is an error unless a parameter value is explicitly provided before solving the initialization problem.
In this case, a tool may give a diagnostic message when translating a simulation model to inform the user that additional parameter values must be provided later.
If the \lstinline!start!-attribute has a value, it can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
```
Like this.

Reply:
I get the point, wouldn't it be too much scope creep to try to switch terminology as part of this PR?